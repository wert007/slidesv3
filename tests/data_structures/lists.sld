func main() {
    let l: List<string> = new List();
    let a = 'he';
    for _ in 0..95 {
        l.add(a);
    }
    print(l);
}