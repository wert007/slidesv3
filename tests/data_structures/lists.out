[ he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he, he ]
5
5
[ hello, hello, hello, hello, hello ]
= 0
